tests/vppoutputencode.cpp

```diff
@@ -34,6 +34,7 @@ EncodeParams::EncodeParams()
     , numRefFrames(1)
     , idrInterval(0)
     , codec("")
+    , enableFrameLatency(0)
```

Comment:
better name it as printLatency or log Latency. enableFrameLatency maybe misleading